gensim/sklearn_api/d2vmodel.py

```diff
@@ -145,6 +145,11 @@ def __init__(self, dm_mean=None, dm=1, dbow_words=0, dm_concat=0, dm_tag_count=1
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
 
+    def _get_first(some_iterable):
+        """Returns first element of some iterable."""
```

Comment:
PEP257: `Return X`, not `Returns X` (imperative mode).

Reply:
Fixed